### PR TITLE
Add cards for VBS WH production for kW = -1

### PR DESCRIPTION
**Pull request by jkguiang**
Adding MadGraph cards for VBS WH production where kappa_W = -1; this is the signal sample for an analysis targeting this process. A detailed summary of this analysis can be found [here](https://indico.cern.ch/event/1201779/contributions/5088549/attachments/2527205/4347174/vbswh_pag_update.pdf).

Finally, the MadGraph model `sm_mkw` used can be found here:
```
/afs/cern.ch/user/j/jguiang/public/models/sm_mkw.tar.gz
```

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBSWH_mkW_Inclusive_4f_LO/VBSWH_mkW_Inclusive_4f_LO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import model sm
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBSWH_mkW_Inclusive_4f_LO/VBSWH_mkW_Inclusive_4f_LO_proc_card.dat:1]: the first line can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBSWH_mkW_Inclusive_4f_LO/VBSWH_mkW_Inclusive_4f_LO_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBSWH_mkW_Inclusive_4f_LO/VBSWH_mkW_Inclusive_4f_LO_run_card.dat:52]: add "$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty" after line 52 to include pdf replicas

**Comment by DickyChant** [comment]
> 
I fully agree. The cards look fine to me, although I would enlarge the etaj cut a little bit as 5 is close to 4.7.

**Comment by jkguiang** [comment]
@DickyChant thanks for the feedback. I believe the original cut of 5 that I had was a typo. It was originally 6.5, which I have fixed in my latest [commit](https://github.com/cms-sw/genproductions/pull/3287/commits/18143a508edd56432fef979430ec0671d95ff432#diff-ad158c013c201c8f121c0004bcae40e7ce9a7b9a5212ad4d689d454f00892fe6R134).

**Comment by jkguiang** [comment]
Hi @menglu21, just wanted to bump this PR.

**Pull request merged**